### PR TITLE
(CDAP-2081) Use schedules that are far in future to avoid program being triggered

### PR DESCRIPTION
**Pull request by chtyim**
- The test only test whether the schedules get store/retrieve successfully, but never need the program to run

### CODE REVIEW

**Comment by sagarkapare** [comment]
LGTM :+1:

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/schedule/SchedulerServiceTest.java [hunk 1]
```diff
@@ -162,6 +166,17 @@ public void testSimpleSchedulerLifecycle() throws Exception {
...
 
+  /**
+   * Returns a crontab that will get triggered after {@code offset} time in the given unit from current time.
+   */
+  private static String getCron(long offset, TimeUnit unit) {
+    Calendar calendar = Calendar.getInstance();
+    calendar.setTimeInMillis(System.currentTimeMillis() + unit.toMillis(offset));
+    return String.format("%s %s %s %s *",
+                         calendar.get(Calendar.MINUTE), calendar.get(Calendar.HOUR_OF_DAY),
+                         calendar.get(Calendar.DAY_OF_MONTH), calendar.get(Calendar.MONTH) + 1);
```
Comment (jwang47) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/schedule/SchedulerServiceTest.java]: It's triggered current time + offset + 1 month?

Comment (chtyim) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/schedule/SchedulerServiceTest.java]: Calendar returns month from 0-11. Crontab takes 1-12

**Comment by chtyim** [comment]
Build passed: http://builds.cask.co/browse/CDAP-DUT2248-1

**Pull request merged**